Components/FirstRunWizSteps/Step1.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { StyleSheet, Text, TextInput, SafeAreaView } from "react-native";
+import { useFormikContext } from "formik";
+import { FirstRunValues, SettingsFormValues } from "../../customTypes";
+import Errors from "../Errors";
+import { formatPhoneNumber } from "../../Helpers/helpers";
+
+export default function Step1() {
+    const { values, errors, handleChange, setFieldValue } =
+        useFormikContext<FirstRunValues>();
+    console.log(errors);
+    return (
+        <SafeAreaView style={styles.container}>
+            <Text style={styles.header}>Name</Text>
+
+            <TextInput
+                onChangeText={handleChange("name")}
+                autoCapitalize="words"
+                value={values.name}
+                style={styles.input}
+            ></TextInput>
+
+            <Errors errorMessage={errors.name} />
+
+            <Text style={styles.header}>Email </Text>
+
+            <TextInput
+                onChangeText={handleChange("email")}
+                autoCapitalize="none"
+                value={values.email}
+                style={styles.input}
+            ></TextInput>
+
+            <Errors errorMessage={errors.email} />
+
+            <Text style={styles.header}>Phone Number </Text>
+
+            <TextInput
+                onChangeText={(text) => {
```

Comment:
You could clean this up to one line. Something like this:
`onChangeText={ (text) => setFieldValue("phone", formatPhoneNumber(text)) }`